_pages/projects/2020-auto-indy.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: project_extended
+title: Indy Autonomous Challenge
+permalink: /projects/2020-auto-indy
+image: /assets/images/projects/2020/project-auto-indy.jpg
+leaders: Nayana Suvarna, Andrew Saba, Joshua Spisak, and Prem Bharatia
```

Comment:
If I was being really picky I'd suggest maintaining the same order as above. Nayana Suvarna, Prem Bharatia, Andrew Saba, Joshua Spisak

Reply:
Lol but this is a super-nitpick. It doesn't matter.

Reply:
Fixed